tests/test_convert.py

```diff
@@ -28,6 +28,12 @@ def fulton_precincts(get_fixture):
         fixture = f.read()
     return fixture
 
+@pytest.fixture
+def gwinnett_precincts(get_fixture):
+    path = os.path.join(FIXTURE_DIR, "gwinnett_precincts_11-3.xml")
```

Comment:
Could we pare down this fixture to just a handful of the contests or just the ones that you have unit test assertions on?

Reply:
yep, good idea